.github/workflows/test.yml

```diff
@@ -1033,7 +1040,7 @@ jobs:
           CI: true
           SKIP_GIT: true
           GITHUB_CONTEXT: ${{ toJson(github) }}
-          NODE_OPTIONS: '--max-old-space-size=8096'
+          NODE_OPTIONS: '--max-old-space-size=10240' # allow Node.js to allocate at most 10gb of heap
```

Comment:
This is about the `CLIENT (memory tests)` job, so, it looks unrelated to the current PR. What is it solving?

Reply:
Random flakiness on Windows that was happening on this PR (although not caused by this PR)